src/configs/shared.ts

```diff
@@ -5,31 +5,18 @@ export const COLORS = {
 export const DEFAULT_BOT_DELAY = 100; // 100ms
 
 /**
- * Timestamps for unassigning features. Not sure this is the best standard for our feature.
- * but it would be fine at least in terms of standarization.
- *
- * ISO 8601(https://en.wikipedia.org/wiki/ISO_8601) is a set of standardized date and time formats
- * in an attempt to tame every programmer's favorite challenge.
- * Durations represent the amount of time between two dates or times.
- *
- * [Format]
- * Y = years
- * M = months
- * W = weeks
- * D = days
- * T = delineator between dates and times, necessary to disambiguate between months and minutes
- * H = hours
- * M = minutes
- * S = seconds
- *
- * [Example]
- * P3Y6M4DT12H30M5S (3 years, 6 months, 4 days, 12 hours, 30 minutes, and 5 seconds)
- * P3DT12H (3 days and 12 hours)
- * P1M (1 month)
- * PT1M (1 minute)
- * PT0S (0)
- * P0.5Y (1/2 a year)
- * PT1M3.025S (1 minute and 3025 milliseconds)
+ * ms('2 days')  // 172800000
+ * ms('1d')      // 86400000
+ * ms('10h')     // 36000000
+ * ms('2.5 hrs') // 9000000
+ * ms('2h')      // 7200000
+ * ms('1m')      // 60000
+ * ms('5s')      // 5000
+ * ms('1y')      // 31557600000
+ * ms('100')     // 100
+ * ms('-3 days') // -259200000
+ * ms('-1h')     // -3600000
+ * ms('-200')    // -200
```

Comment:
Wouldn't negative break things? We should probably throw an error for negative values.

Reply:
good catch

Reply:
I set approval assuming that you will finish this.

Reply:
https://github.com/ubiquity/bounty-bot/pull/105/commits/1081d8f6ae7c55fee84e336b2ccbbcca54837061